cfme/tests/pod/test_appliance_crud.py

```diff
@@ -325,6 +330,98 @@ def ipa_user(temp_pod_appliance, ipa_auth_provider):
     appliance.server.login_admin()
 
 
+@pytest.fixture
+def aws_provider(temp_pod_appliance):
+    from cfme.utils.providers import get_crud
+    prov = get_crud('ec2west')
```

Comment:
**Question** Is there a reason this is hardcoded in? If the name changes, the test will break.